tests/test_machine.py

```diff
@@ -1,69 +1,68 @@
 # SPDX-License-Identifier: LGPL-2.1+
 
-import os
+import unittest
 from subprocess import TimeoutExpired
+from sys import exc_info
 
-import pytest
-
-import mkosi.machine as machine
 from mkosi.backend import MkosiException
+from mkosi.machine import Machine
 
 
-pytestmark = [
-    pytest.mark.integration,
-    pytest.mark.parametrize("verb", ["boot", "qemu"]),
-    pytest.mark.skipif(os.getuid() != 0, reason="Must be invoked as root.")
-]
-
-
-def test_simple_run(verb: str) -> None:
-    with machine.Machine([verb]) as m:
-        p = m.run(["echo", "This is a test."])
-        assert "This is a test." == p.stdout.strip("\n")
-
-    assert m.exit_code == 0
-
+class MkosiMachineTest(unittest.TestCase):
+    reboot: bool
+    args: str
+    machine: Machine
 
-def test_wrong_command(verb: str) -> None:
-    # First tests with argument check = True from mkosi.backend.run(), therefore we see if an exception is raised
-    with machine.Machine([verb]) as m:
-        with pytest.raises(MkosiException):
-            m.run(["NonExisting", "Command"])
-        with pytest.raises(MkosiException):
-            m.run(["ls", "NullDirectory"])
+    def __init_subclass__(cls, /, args: str, reboot: bool = True) -> None:
```

Comment:
Why is `args` a string here? Shouldn't it be a list of strings?

Reply:
This was actually a typo.